srtcore/core.cpp

```diff
@@ -8379,6 +8386,28 @@ void CUDT::processCtrlAck(const CPacket &ctrlpkt, const steady_clock::time_point
     leaveCS(m_StatsLock);
 }
 
+int CUDT::processLossSeq(const int seqno, const time_point& time_nak, const time_point& time_now)
+{
+    const int offset = CSeqNo::seqoff(m_iSndLastDataAck, seqno);
+    steady_clock::time_point tsLastRexmit;
+    m_pSndBuffer->getPacketTime(offset, tsLastRexmit);
```

Comment:
Assignment to `tsLastRexmit` is here invisible, please use `m_pSndBuffer->getPacketTime(offset, (tsLastRexmit));`. An even better solution though would be if you simply return the value that you assign to `tsLastRexmit`.